Library/Formula/flif.rb

```diff
@@ -0,0 +1,24 @@
+class Flif < Formula
+  desc "Free Lossless Image Format"
+  homepage "http://flif.info"
+  url "https://github.com/FLIF-hub/FLIF/archive/v0.1-alpha.tar.gz"
+  version "0.1-alpha"
```

Comment:
Unless there's a stable release this should probably live in `homebrew/homebrew-devel-only`. I'm happy to review it here until then, though.